src/App.js

```diff
@@ -1,11 +1,39 @@
-import "./App.css";
-import PeriodicTable from "./Components/PeriodicTable.js";
+import PeriodicTable from './Components/PeriodicTable.js';
+import styled from 'styled-components';
 
 export default function App() {
   return (
-    <div className="App">
-<h1>Periodic Table of Elements</h1>
-<PeriodicTable />
-    </div>
+    <StyledBody>
+      <header>
```

Comment:
You have created a styled component (StyledHeader) for the header element, however you do not use it here 🤔Anyhow, as you only have an h1 in your header, you do not need the header element. It is rather used to group several elements.
```suggestion
      <header>
```